bootstrap.yml

```diff
@@ -969,6 +969,8 @@ packages:
         - '-DLLVM_HOST_TRIPLE=@OPTION:arch-triple@'
         # Disable linking against ncurses, which we do not build with -fPIC.
         - '-DLLVM_ENABLE_TERMINFO=OFF'
+        # Suppress developer warnings
+        - '-Wno-dev'
```

Comment:
Why do we do that?

Reply:
Because it spams the console during LLVM configuration and we aren’t LLVM developers.